zips/zip-0234.md

```diff
@@ -16,138 +17,119 @@ License: BSD-2-Clause
-would distribute that increase starting in the immediately following block and
-subsequent blocks. Because this distribution mechanism has an exponential
-decay, such increases will be spread out in miniscule amounts to future blocks
-over a long time period. This issuance mechanism thus provides a way for
-potential increases or decreases of issuance while constraining those changes
-to be small on a short time scale to avoid unexpected disruptions. 
-
-Additionally, the current Bitcoin-style issuance does not take into account the
-current balance of `ZsfBalanceAfter(h)`. If [#draft-zsf]_ were to activate
-without a change to the issuance mechanism, then some funds would never be
-disbursed after they are deposited back into the ZSF.
+This new NSM-based issuance scheme solves these problems by ensuring a more
+consistent and predictable rate of coin issuance, while preserving the core
+aspects of Zcash's issuance policy and the 21 million coin cap. At the same
+time, it introduces the first mechanism to recreate and distribute ZEC that has
+been removed from circulation, as well as unclaimed transaction fees, across
+future block subsidies.
 
 # Requirements
 
 Smoothing the issuance curve is possible using an exponential decay formula
 that satisfies the following requirements:
 
-## Issuance Requirements
-
-1. The issuance can be summarised into a reasonably simple explanation
+1. The issuance can be summarized into a reasonably simple explanation
 2. Block subsidies approximate a continuous function
-3. If there are funds in the ZSF, then the block subsidy must be non-zero,
-   preventing any final “unmined” zatoshis
+3. If the Money Reserve is greater than 0, then the block subsidy must be
+   non-zero, preventing any final "unmined" zatoshis
 4. For any 4 year period, all paid out block subsidies are approximately equal
-   to half of the ZSF at the beginning of that 4 year period, if there are no
-   deposits into the ZSF during those 4 years
+   to half of the Money Reserve at the beginning of that 4 year period, if no
+   ZEC is burned during those 4 years
+5. Decrease the short-term impact of the deployment of this ZIP on block reward
+   recipients, and minimize the potential reputation risk to Zcash of changing
+   the block reward amount.
 
-TODO daira: add a requirement that makes the initial total issuance match the previous total issuance  
+TODO daira: add a requirement that makes the initial total issuance match the previous total issuance
 
 # Specification
 
-## Goals
-
-We want to decrease the short-term impact of the deployment of this ZIP on
-block reward recipients, and minimise the potential reputational risk to Zcash
-of changing the block reward amount.
-
-## Constants
+## Definitions
 
-Define constants:
+`BLOCK_SUBSIDY_FRACTION` = 4126 / 10,000,000,000 or `0.0000004126`
 
-“`BLOCK_SUBSIDY_FRACTION`” = 4126 / 10,000,000,000 or `0.0000004126`
+`DEPLOYMENT_BLOCK_HEIGHT` = 2726400
```

Comment:
```suggestion
`DEPLOYMENT_BLOCK_HEIGHT` = TBD
```
This height can be no less than the NU7 activation; it could be afterward.

Reply:
We haven't had time to calculate a precise height. However, with this edit I'm aiming to present something that's relatively unambiguous for the purposes of semantic stability:

```suggestion
`DEPLOYMENT_BLOCK_HEIGHT` = `TBD`

The block height will be chosen by the following criteria:

- It is after NU7 activation height.
- It is calculated to be the lowest height after the the second halving at which the NSM issuance would be less than the current BTC-style issuance _neglecting_ any burnt ZEC (ie. assuming the amount of ZEC burnt is exactly 0).

This selection is intended to achieve Key Objective 6 while still being a constant height. An alternative would be to have a dynamic "latch" style activation which would calculate the activation height by testing the "less than" conditional with every block after the second halving. We prefer the pre-defined constant height parameter to give everyone more _time_ certainty at the cost of _issuance level_ certainty. The difference in up-front calculation versus dynamic calculation is whether or not burns are accounted for (since future burns cannot be calculated up-front). This means with the pre-defined constant parameter approach, issuance will jump _up_ some amount at activation. This amount should be equivalent to all ZEC burnt prior to that height times `BLOCK_SUBSIDY_FRACTION`. For example, if a total of 100,000 ZEC were burnt prior to the pre-defined constant activation height, then at activation the issuance would be larger than BTC-style issuance by `100_000 ZEC * BLOCK_SUBSIDY_FRACTION` which we calculate equals `0.04126 ZEC`. This example is chosen to demonstrate that a very large burn amount (much larger than expected) would elevate issuance by a relatively small amount. For this reason, we believe a pre-defined constant is a better approach to achieving Key Objective 6 than a "dynamic latch" logic because it is so much simpler to implement and reason about.
```

I'm not sure if that level of discussion about "dynamic latching" is necessary. It could be removed to just specify the pre-defined constant height in a future revision without altering semantic stability, in my understanding.